vnext/Microsoft.ReactNative/Fabric/ImageComponentView.cpp

```diff
@@ -24,8 +24,8 @@ ImageComponentView::ImageComponentView(winrt::Microsoft::ReactNative::ReactConte
   static auto const defaultProps = std::make_shared<facebook::react::ImageProps const>();
   m_props = defaultProps;
 
-  m_onLoadEndToken = m_element->OnLoadEnd([=](const auto &, const bool &succeeded) {
-    if (succeeded) {
+  m_onLoadEndToken = m_element->OnLoadEnd([=](const auto &, const winrt::hstring &error) {
+    if (error.empty()) {
       std::static_pointer_cast<const facebook::react::ImageEventEmitter>(m_eventEmitter)->onLoad();
     } else {
```

Comment:
>else  [](http://example.com/codeflow?start=6&length=5)

Given a non-empty error message, should it get passed along somewhere?

Reply:
Hmm I don't know React Native internals well enough to understand where this goes, but am assuming these are internal events that don't cross to JS side and the listeners just need to know of the fact the error happened. ImageEventEmitter interface doesn't support passing messages to onError.